tests/unit/test_results.py

```diff
@@ -0,0 +1,170 @@
+import copy
+import datetime
+import random
+
+import responses
+
+from intezer_sdk.analyses_history import AnalysesHistory
+from intezer_sdk.api import get_global_api
+from intezer_sdk.results import Results
+from tests.unit.base_test import BaseTest
+
+BASE_DATA = AnalysesHistory._data_analyses_history(
```

Comment:
module parameters should be consts, but in tests we regularly changing these, so instead of using `copy`, move the definition to the setUp function so every test will have its own new data